target/linux/ath79/nand/base-files/etc/board.d/02_network

```diff
@@ -20,6 +20,11 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "2:lan:2" "3:lan:1" "1:wan"
 		;;
+	netgear,r6100)
+		ucidef_set_interfaces_lan_wan "eth0.1" "eth1"
```

Comment:
LAN will be set automatically by the switch command, so 
ucidef_set_interface_wan "eth1"
will be enough for the first line.